x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/diff_components/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './index_patterns';
+export * from './field_diff';
```

Comment:
Let's export also the `header_bar` and import it from the `index` where it's used